packages/orbit-components/src/Popover/index.tsx

```diff
@@ -152,13 +147,15 @@ const Popover = ({
 
   return (
     <>
-      <StyledPopoverChild
+      {/* eslint-disable-next-line jsx-a11y/no-static-element-interactions */}
```

Comment:
it can't be focusable or at least if the Button is used inside, so it have onClick and keyboard event and it works because of bubbling, but should have tabIndex={-1}, so it won't have double focus (the need to tab twice, because there is a button for opening usually). I agree we should do something with this ;)

Reply:
Yes, I'd say we can address this later, let's just not forget about it 🙏